pacfetcher_test.go

```diff
@@ -18,7 +18,7 @@ import (
 )
 
 func pacjsHandler(pacjs string) http.HandlerFunc {
-	return func(w http.ResponseWriter, r *http.Request) { w.Write([]byte(pacjs)) }
+	return func(w http.ResponseWriter, r *http.Request) { w.Write([]byte(pacjs)) } // nolint:errcheck
```

Comment:
I think `_, _ := w.Write(...)` also gets rid of the linter warning (same for all the others below).
Did you intentionally choose the `// nolint: ...` ?

Reply:
Sam said a similar thing. I just wanted to silence the warnings because i wanted to see only lint warnings from my code. The comment says I am silencing the warnings. The underscores says I am ignoring the warnings. Ignoring them should be a deliberate decision, one that I wasn't going to make. With the comment, it is easy to grep for that and revisit if these are to be fixed properly by doing something with the error. The other form is not as easy to grep for.

I'll leave it up to Sam what he wants done with this, and then tell him they're his lint warnings anyway :)

Reply:
If you're going to start using a linter, can you document this in something like CONTRIBUTING.md? I'll go through the lint errors in my code and fix them.

Reply:
Better also covered CI - Github actions? Travis?  Beyond the scope of this PR I'd say.
I've created an issue and assigned to myself #20 (including creation of CONTRIBUTING.md)

Reply:
Also @anzdaddy I don't think `golangci-lint 1.17` with "strict" settings complains about `_, _ := ...`.
"strict" being:
```yaml
issues:
  exclude-use-default: false
linters:
  enable-all: true
```

Reply:
```yaml
linters-settings:
  errcheck:
    check-blank: true
```

Reply:
💥  :)

Reply:
The linter settings to use should be discussed in a relevant issue, not here. It will get lost here.

@samuong I will remove the //nolint commit from this PR and re-work my code where I added one so the topic of linting disappears from this PR, and it can be covered in #20 (Setup CI) and your PR. To do so, I'll squash all my fixes so far into the relevant commits and force push the branch with the reduced set of commits.